client/src/components/public/calendar/EventType.vue

```diff
@@ -0,0 +1,380 @@
+<template>
+    <div class="main">
+        <div class="event-type-container">
+            <VRow>
+                <VCol :class="colEventInfoClass">
+                    <DatailLayout>
+                        <div class="event-info">
+                            <VIcon dark color="primary"
+                                >mdi-clock-outline</VIcon
+                            >
+                            {{ meetingDuration }} {{ lang.ET_MIN }}
+                        </div>
+                        <div class="event-info">
+                            <VIcon dark color="primary">mdi-map-marker</VIcon>
+                            {{ meetingLocation }}
+                        </div>
+                    </DatailLayout>
+                </VCol>
+
+                <VCol class="calendar-container col-sm-12 col-md-6">
+                    <div class="calendar-content">
+                        <h3>{{ lang.ET_SELECT_DATE_AND_TIME }}</h3>
+                        <VDatePicker
+                            v-model="date"
+                            min="2020-08-10"
+                            :first-day-of-week="1"
+                            :weekday-format="getDay"
+                            :header-date-format="dateFormat"
+                            no-title
+                            :width="calendarWidth"
```

Comment:
date picker has property "full-width", so you can create a wrapper over date picker, constraint it with widh you need, and manipulate width with css. It should help you get rid of calendarWidth